hack/test_errata.py

```diff
@@ -0,0 +1,738 @@
+import datetime
+import errata
+import github
+import http
+import json
+import os
+import re
+import tempfile
+import unittest
+import urllib
+from collections import namedtuple
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+
+GithubUserMock = namedtuple("GithubUserMock", "login")
+GithubLabelMock = namedtuple("GithubLabelMock", "name")
+class GithubPRMock:
+    def __init__(self, user, title, labels=[], number=0, body="", url="", html_url=""):
+        self.user = user
+        self.title = title
+        self.labels = labels
+        self.number = number
+        self.body = body
+        self.url = url
+        self.html_url = html_url
+        self.create_issue_comment = MagicMock()
+        
+    def __eq__(self, other):
+        if not isinstance(other, GithubPRMock):
+            return False
+        return      self.user == other.user     \
+                and self.title == other.title   \
+                and self.labels == other.labels \
+                and self.number == other.number \
+                and self.body == other.body     \
+                and self.url == other.url       \
+                and self.html_url == other.html_url
+
+
+class ExtractErrataNumberFromBodyTest(unittest.TestCase):
+    def test_valid_url(self):
```

Comment:
```suggestion
def test_url(self):
```
This test case has both valid and invalid urls, so lets have a single test for those

Reply:
I need to be more explicit and I'll need to definitely work on naming variables, functions, etc.

By valid URL I meant URL which starts with corresponding `ERRATA_MARKER` in the `errata.py`:
`https://errata.devel.redhat.com/advisory/`

I'll change the name of the function and I'll add the tests as well.

Thanks